_posts/2018-11-19-install-a-client-locally-with-fake-secondaries.adoc

```diff
@@ -105,27 +103,10 @@ localtest/
 include::https://raw.githubusercontent.com/advancedtelematic/aktualizr/{aktualizr-version}/config/sota-local.toml[]
 ----
 
-[source,role="secondary",subs="attributes"]
-.virtualsec1.json
-----
-include::https://raw.githubusercontent.com/advancedtelematic/aktualizr/{aktualizr-version}/config/secondary/virtualsec.json[]
-----
-
 [source,role="secondary"]
```

Comment:
Needs the `,subs="attributes"` part--that's what allows the `{aktualizr-version}` inside the code block to be processed, instead of rendered literally.

Reply:
corrected